tdrs-backend/tdpservice/settings/test.py

```diff
@@ -17,3 +17,6 @@ class Local(Common):
     EMAIL_HOST = "localhost"
     EMAIL_PORT = 1025
     EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+    # Always use Localstack during tests
+    USE_LOCALSTACK = True
```

Comment:
This override exists to prevent accidentally running unit tests against a live S3 if a developer is running the app locally with localstack turned off.

Reply:
Smart default!